app/helpers/application_helper.rb

```diff
@@ -44,4 +46,8 @@ def paperclip_full_url(upload)
     "#{Rails.application.config.action_controller.relative_url_root}" \
       "#{upload.url}"
   end
+
+  def env?(var)
```

Comment:
Yeah, I wasn't super happy with putting it here, it just seemed like the easiest place to put it to get access to it everywhere. That being said, just because it's easiest doesn't mean it's the best! The base initializer sounds like a much better idea